mesonbuild/compilers/compilers.py

```diff
@@ -717,7 +717,8 @@ def has_header_symbol(self, hname: str, symbol: str, prefix: str,
 
     def run(self, code: 'mesonlib.FileOrString', env: 'Environment', *,
             extra_args: T.Union[T.List[str], T.Callable[[CompileCheckMode], T.List[str]], None] = None,
-            dependencies: T.Optional[T.List['Dependency']] = None) -> RunResult:
+            dependencies: T.Optional[T.List['Dependency']] = None,
+            run_cwd: T.Optional[str] = None) -> RunResult:
```

Comment:
Commit message: what is "1sftworkdir"? :D

Reply:
fixed